src/components/appbar/DBAppBar.js

```diff
@@ -84,19 +84,17 @@ const DBAppBar = ({
       ) : null}
 
       {loc ? (
-        <HStack space="1" alignItems="center">
-          {locationIcon}
-          <Text color="white" fontSize={fp(2)}>
-            {location}
-          </Text>
-          <IconButton
-            icon={editIcon}
-            _pressed={{
-              backgroundColor: theme.colors.secondary[200],
-            }}
-            onPress={() => navigation.navigate('EditLocation')}
-          />
-        </HStack>
+        <TouchableOpacity
+          activeOpacity={0.4}
+          onPress={() => navigation.navigate('EditLocation')}>
```

Comment:
use Const for onPress
and call that const here

Reply:
okay i am changing it sir